WalletWasabi/BitcoinCore/RpcClientBase.cs

```diff
@@ -99,6 +99,11 @@ public virtual IRPCClient PrepareBatch()
 			return new RpcClientBase(Rpc.PrepareBatch());
 		}
 
+		public virtual Task<RPCResponse> SendCommandAsync(RPCOperations operations, params object[] parameters)
```

Comment:
I am not the code owner of the RPC code but isn't this request too general here? @lontivero what was/is your concept here in this class?

Reply:
About `SendCommandAsync` what is the reasoning behind hiding it?
To void it I'd have to put the complex code in the implementation and not as an extension method and still it will be challenging to mock the response without

Reply:
`IRpcClient` and its hierarchy are for abstracting the access to a bitcoin client. `SendCommandAsync` is the opposite, in fact, it is there just to provide access to those some methods are not implemented otherwise it should be private in nbitcoin. 

It is interesting what you say about where to add the logic because the extension methods were created because there was no other way to add functionality to the rpc client but that is not longer true. So, if I have a class Person with two properties FirstName and LastName and I want to have a GetFullName() method, I can add that method to the class itself or create an extension method. Where should I do?

Reply:
Fair point, I am not aware to the history of decisions, so in that case the RPC impl should probably have more logic in it at the expense of the extension method. fine with me

Reply:
I decided to make a new class RpcParser seems cleaner that way on the Rpc impl.